### PR TITLE
[GUI Editor] UI Update

### PR DESCRIPTION
**Pull request by darraghjburke**
Improves the UI across the GUI editor in various places:

* Adds arrows to canvas resolution and shadow offsets
* Fixes the spacing of the properties panel - units now form a vertical line
* Updates the font file: font faces are now all Acumin
* Left grab and right grab are now invisible and exist at the edge of the left and right panels
* Fixes some linting issues

### CODE REVIEW

**Comment by RaananW** [comment]
There are build issues in the gui-editor project

File: packages/tools/guiEditor/src/diagram/workbench.tsx [hunk 1]
```diff
@@ -668,7 +668,7 @@ export class WorkbenchComponent extends React.Component<IWorkbenchComponentProps
         return true;
     }
 
-    onMove(evt: React.PointerEvent) {
+    onMove() {
```
Comment (carolhmj) [inline packages/tools/guiEditor/src/diagram/workbench.tsx:671]: evt is used just below on line 684, so it shouldn't be removed.

Comment (darraghjburke) [reply packages/tools/guiEditor/src/diagram/workbench.tsx:671]: Nice catch, thank you!

**Review by carolhmj — APPROVED** [review]

**Comment by RaananW** [comment]
Just making sure - is the color picker working correctly int the other packages?

**Pull request merged**